### PR TITLE
TW-795: inkwell in web version are not necessary anymore

### PR DESCRIPTION
**Pull request by imGok**
more details on https://github.com/Sorunome/matrix_link_text/pull/5

### CODE REVIEW

File: lib/link_text.dart [hunk 1]
```diff
@@ -310,28 +313,34 @@ TextSpan LinkTextSpans(
...
-          textSpans.add(
-            LinkTextSpan(
-              text: linkText,
-              style: linkStyle,
-              url: uri,
-              onLinkTap: launchUrlIfHandler,
-            ),
-          );
-        }
+        // if (kIsWeb) {
```
Comment (drminh2807) [inline lib/link_text.dart]: you can remove it

Comment (imGok) [reply lib/link_text.dart]: done

**Comment by imGok** [comment]
@hoangdat can we merge this one ?

**Review by drminh2807 — APPROVED** [review]

**Review by hoangdat — APPROVED** [review]

File: lib/link_text.dart [hunk 2]
```diff
@@ -15,6 +14,11 @@ import 'schemes.dart';
 import 'tlds.dart';
 
 typedef LinkTapHandler = void Function(Uri);
+typedef TextSpanBuilder = TextSpan Function(
+  String? text,
```
Comment (sherlockvn) [inline lib/link_text.dart:18]: if it's optional, why don't you put it inside { }, so that we don't have to write null again and again ?

Comment (imGok) [reply lib/link_text.dart:18]: it's related to @drminh2807 's feature

**Pull request merged**